packages/manager/src/features/CloudPulse/Alerts/AlertsLanding/AlertsLanding.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {
+  Redirect,
+  Route,
+  Switch,
+  useLocation,
+  useRouteMatch,
+} from 'react-router-dom';
+
+import { Box } from 'src/components/Box';
+import { Paper } from 'src/components/Paper';
+import { TabLinkList } from 'src/components/Tabs/TabLinkList';
+import { Tabs } from 'src/components/Tabs/Tabs';
+import { useFlags } from 'src/hooks/useFlags';
+
+import { AlertDefinitionLanding } from './AlertsDefinitionLanding';
+
+export const AlertsLanding = React.memo(() => {
+  const flags = useFlags();
+  const { url } = useRouteMatch();
+  const { pathname } = useLocation();
+  const tabs = React.useMemo(
```

Comment:
Can we add a type parameter to this?
```suggestion
  const tabs = React.useMemo<Tab[]>(
```

Also add to the imports:

```
import type { Tab } from 'src/components/Tabs/TabLinkList';
```

Reply:
@hkhalil-akamai , The accessible property added to the tabs here for the visibility might not be the same as the Tab from the TabLinkList , should we create a custom type for this here or is there any other way to do this ?

Reply:
Oh good callout Hussain. Yes we want to make changes here. Give me a few and I'll point them out

Reply:
We don't want to change the Tab interface, so I suggest you create a new interface that's something like:
```
interface AlertsTab {
  isEnabled: boolean;
  tab: Tab;
}
```

```
const tabs = React.useMemo<AlertsTab[]>(
    () => [
      {
        isEnabled: Boolean(flags.aclpAlerting?.alertDefinitions),
        tab: {
          routeName: `${url}/definitions`,
          title: 'Definitions',
        },
      },
      // Add more tabs here as needed
    ],
    [url]
  );
```

Then you can filter based on that
```
const accessibleTabs = React.useMemo(
    () =>
      tabFlags
        .filter((tabFlag) => tabFlag.isTabEnabled)
        .map((tabFlag) => tabFlag.tab),
    [tabFlags, flags]
  );
```

Reply:
Yea that makes sense. I updated my snippets